files/scripts/write_standby.py

```diff
@@ -36,9 +36,18 @@ def config_db(self):
         Returns config DB connector.
         Initializes the connector during the first call
         """
-        if self.config_db_connector is None:
-            self.config_db_connector = ConfigDBConnector()
-            self.config_db_connector.connect()
+        retries = 3
+        while self.config_db_connector is None and retries > 0:
+            try:
+                self.config_db_connector = ConfigDBConnector()
```

Comment:
If we set `use_unix_socket_path=True`, would that avoid this issue entirely and remove the need for retries?

Reply:
You are right, this is a better way.